tdrs-backend/tdpservice/parsers/schema_defs/ssp/m5.py

```diff
@@ -152,22 +152,21 @@
                 Field(
                     item="14",
                     name="DATE_OF_BIRTH",
-                    friendly_name="date of birth",
+                    friendly_name="Date of Birth",
                     type="string",
                     startIndex=20,
                     endIndex=28,
                     required=True,
-                    validators=[validators.intHasLength(8),
-                                validators.dateYearIsLargerThan(1900),
-                                validators.dateMonthIsValid(),
-                                validators.dateDayIsValid()
-                                ],
+                    validators=[
```

Comment:
@elipe17 @victoriaatraft same questions as above.

Reply:
@ADPennington I will follow up with @EricYagerPersonalRepo on these right now. I know we had some conflicts that he helped us with.